2025-05-28 10:47:46,328 - INFO - 简化VNC代理服务器启动在端口 5900
2025-05-28 10:47:50,909 - INFO - 新客户端连接: ('127.0.0.1', 46055)
2025-05-28 10:47:50,909 - INFO - 为客户端 ('127.0.0.1', 46055) 创建新VNC会话
2025-05-28 10:47:53,792 - INFO - 新客户端连接: ('127.0.0.1', 46064)
2025-05-28 10:47:53,793 - INFO - 有活跃会话，新客户端 ('127.0.0.1', 46064) 等待决策
2025-05-28 10:47:57,706 - INFO - 断开当前会话: ('127.0.0.1', 46055)
2025-05-28 10:47:57,706 - INFO - 数据转发异常: 客户端->VNC - [WinError 10053] 你的主机中的软件中止了一个已建立的连接。
2025-05-28 10:47:57,706 - INFO - 数据转发异常: VNC->客户端 - [WinError 10053] 你的主机中的软件中止了一个已建立的连接。
2025-05-28 10:47:57,706 - INFO - 为客户端 ('127.0.0.1', 46064) 创建新VNC会话
2025-05-28 10:47:58,780 - INFO - 新客户端连接: ('127.0.0.1', 46082)
2025-05-28 10:47:58,781 - INFO - 客户端 ('127.0.0.1', 46082) 在冷却期内，拒绝连接
2025-05-28 10:47:58,785 - INFO - 客户端版本: b'RFB 003.008\n'
2025-05-28 10:47:58,785 - INFO - 已发送RFB拒绝消息: 服务器正被其他用户使用，请稍后再试。
2025-05-28 10:48:34,895 - INFO - 清理会话: ('127.0.0.1', 46064)
2025-05-28 10:48:34,896 - INFO - 数据转发异常: 客户端->VNC - [WinError 10053] 你的主机中的软件中止了一个已建立的连接。
2025-05-28 10:48:34,896 - INFO - 清理会话: ('127.0.0.1', 46064)
2025-05-28 10:48:34,896 - INFO - 数据转发异常: VNC->客户端 - [WinError 10053] 你的主机中的软件中止了一个已建立的连接。
2025-05-28 11:14:54,380 - ERROR - An error occurred when calling message handler
Traceback (most recent call last):
  File "pystray\_win32.py", line 412, in _dispatcher
  File "pystray\_win32.py", line 224, in _on_notify
  File "pystray\_base.py", line 328, in inner
  File "pystray\_base.py", line 453, in __call__
  File "vnc_proxy.py", line 436, in quit_application
SystemExit: 0
